### DISCUSSION TITLE
Debugging Output

### DISCUSSION THREAD

**Discussion by seandenigris** [hpi-swa/smalltalkCI#650] [General]
I noticed the config variables are dumped to output on errors, but what about others e.g. those with pharo_ gt_ prefixes? Why are those not dumped? Should they be? It seems like it would be useful...

**Comment by fniephaus** [comment]
I guess no one has thought about that yet. Feel free to open a PR to fix this ;)

**Comment by seandenigris** [comment]
Created issue https://github.com/hpi-swa/smalltalkCI/issues/663. I'll try to have a look when time allows
